library/libraries.js

```diff
@@ -1171,5 +1171,15 @@ var d41d8cd98f00b204e9800998ecf8427e_LibraryDetectorTests = {
             }
             return false;
         }
+    },
+    'boomerang.js': {
+        icon: 'icon_48', // currently has no icon
+        url: 'https://soasta.github.io/boomerang/doc/',
+        test: function(win) {
+            if (win.BOOMR) {
```

Comment:
In addition to the check for `window.BOOMR`, please also use one of the non-version properties from the `BOOMR` object when checking for existence of boomerang.